src/wp-includes/block-supports/duotone.php

```diff
@@ -0,0 +1,392 @@
+<?php
+/**
+ * Duotone block support flag.
```

Comment:
As discussed in https://github.com/WordPress/gutenberg/pull/26752#issuecomment-842519553 with @ajlende, we need to include the license from `TinyColor` library.